seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/core/base/command/BaseTaskExecuteCommand.java

```diff
@@ -96,7 +95,8 @@ protected final void close(List<? extends Plugin<E>>... plugins) {
     protected void showAsciiLogo() {
         String printAsciiLogo = System.getenv("SEATUNNEL_PRINT_ASCII_LOGO");
         if ("true".equalsIgnoreCase(printAsciiLogo)) {
-            AsciiArtUtils.printAsciiArt(Constants.LOGO);
```

Comment:
Remove unused code?

https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/core/base/utils/AsciiArtUtils.java
https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-core/seatunnel-core-starter/src/main/java/org/apache/seatunnel/core/starter/utils/AsciiArtUtils.java

Reply:
ok ,that sounds good;